### ISSUE TITLE
minute: AttributeError: 'dict' object has no attribute 'inject_to'

### ISSUE THREAD

**Issue by sopos** [teemtee/tmt#1396]
the plan 
```yaml
/plans/upgrade/aide:
    discover:
        how: fmf
        test:
          - /aide/Sanity/aide-check-sanity
    execute:
        how: upgrade
        url: https://gitlab.cee.redhat.com/oamg/rhel-major-upgrade
        upgrade-path: /paths/el8toel9.supported
    summary: aide upgrade plan
```
used in https://github.com/RedHat-SP-Security/tests causes a traceback.
```
$ tmt -vvvddd run -a plans -n /plans/upgrade provision -h minute -i 1MT-RHEL-8.7.0-20220722.0                                                                                                                                                                                             
/var/tmp/tmt/run-006                                                           

/plans/upgrade/aide                                                            
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/step.yaml'.                                                                                    
    Step data not found.                                                       
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/tests.yaml'.                                                                                   
    Discovered tests not found.                                                
    Using the 'DiscoverFmf' plugin for the 'fmf' method.                       
    status: todo                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/step.yaml'.                                                                                   
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/tests.yaml'.                                                                                  
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/prepare/step.yaml'.                                                                                     
    Step data not found.                                                       
    Using the 'PrepareShell' plugin for the 'shell' method.                                                                                                    
    status: todo                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/prepare/step.yaml'.                                                                                    
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/execute/step.yaml'.                                                                                     
    Step data not found.                                                       
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/execute/results.yaml'.                                                                                  
    Test results not found.                                                    
    Using the 'ExecuteUpgrade' plugin for the 'upgrade' method.                                                                                                
    status: todo                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/execute/step.yaml'.                                                                                    
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/execute/results.yaml'.                                                                                 
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/report/step.yaml'.                                                                                      
    Step data not found.                                                       
    Report step always force mode enabled.                                     
    Clean up workdir '/var/tmp/tmt/run-006/plans/upgrade/aide/report'.                                                                                         
    status: todo                                                               
    Using the 'ReportDisplay' plugin for the 'display' method.                                                                                                 
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/report/step.yaml'.                                                                                     
    Read file '/var/tmp/tmt/run-006/plans/upgrade/aide/finish/step.yaml'.                                                                                      
    Step data not found.                                                       
    Using the 'FinishShell' plugin for the 'shell' method.                                                                                                     
    status: todo                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/finish/step.yaml'.                                                                                     
    discover                                                                   
    workdir: /var/tmp/tmt/run-006/plans/upgrade/aide/discover                                                                                                  
        how: fmf                                                               
        order: 50                                                              
        Run command 'git rev-parse --show-toplevel'.                           
        out: /home/dapospis/test/git/SP_components/tests                       
        directory: /home/dapospis/test/git/SP_components/tests                                                                                                 
        Copy '/home/dapospis/test/git/SP_components/tests' to '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/default/tests'.                                                                                                                                                                                               
        Run command 'git rev-parse --short HEAD'.                              
        out: 97c9df9                                                           
        hash: 97c9df9                                                          
        tests: /aide/Sanity/aide-check-sanity                                  
        Check metadata tree in '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/default/tests/'.                                                              
        summary: 1 test selected                                               
            /aide/Sanity/aide-check-sanity                                     
    status: done                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/step.yaml'.                                                                                   
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/tests.yaml'.                                                                                  
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/discover/run.yaml'.                                                                                    
    provision                                                                  
        how: minute                                                            
        order: 50                                                              
        image: 1MT-RHEL-8.7.0-20220722.0                                       
        flavor: m1.small                                                       
        auto-select-network: False                                             
        skip-prereserve: False                                                 
    finish                                                                     
    workdir: /var/tmp/tmt/run-006/plans/upgrade/aide/finish                                                                                                    
        summary: 0 tasks completed                                             
    status: done                                                               
    Write file '/var/tmp/tmt/run-006/plans/upgrade/aide/finish/step.yaml'.                                                                                     
Traceback (most recent call last):                                             
  File "/usr/bin/tmt", line 8, in <module>
    tmt.cli.main()
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1700, in invoke
    return _process_result(rv)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 1637, in _process_result
    value = ctx.invoke(self._result_callback, value, **ctx.params)
  File "/usr/lib/python3.10/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.10/site-packages/tmt/cli.py", line 384, in finito
    click_context.obj.run.go()
  File "/usr/lib/python3.10/site-packages/tmt/base.py", line 1822, in go
    plan.go()
  File "/usr/lib/python3.10/site-packages/tmt/base.py", line 1040, in go
    step.go()
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/__init__.py", line 142, in go
    phase.go()
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/minute.py", line 249, in go
    self._guest = GuestMinute(data, name=self.name, parent=self.step)
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/__init__.py", line 298, in __init__
    self.load(data)
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/minute.py", line 269, in load
    super().load(data)
  File "/usr/lib/python3.10/site-packages/tmt/steps/provision/__init__.py", line 327, in load
    data.inject_to(self)
AttributeError: 'dict' object has no attribute 'inject_to'
```

Last known working version: tmt-1.15.0-1.20220721121027456355.main.14.g13e948a.fc35

**Comment by lukaszachy** [comment]
Unfortunately we broke API in COPR builds. This is issue in internal "minute" plugin and will be fixed only for 1.16.
See Internal gitlab issue 23 for further info.

In the future we plan to run tests using internal plugins _before_ PR is merged.

**Comment by lukaszachy** [comment]
1.16 was released, Fedora updates are [made](https://bodhi.fedoraproject.org/updates/?packages=tmt).
Once rpms reach stable repos internal package can be released.

Virtualenv / PyPI tmt + internal plugin is working (note that both has to be installed by pip), e.g. `pip3 install --user "git+https://internal_repo"`. So I am closing this ussue.
